packaging/flatpak/generate-flatpak-script.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
`#!/usr/bin/env bash` is preferred, `env` is much more likely to be found even on distros that don't conform to the standard filesystem hierarchy